frameworks/kafka/docs/api-reference.md

```diff
@@ -400,7 +400,16 @@ Note the warning in the output from the commands above. You can change the indic
     {
         "message": ""
     }
-    
+
+# Service Status
```

Comment:
Yes, that would be easier. This way this information doesn't get held up while I/we figure out how best to do this.